### ISSUE TITLE
Protect master branch

### ISSUE THREAD

**Issue by philipnrmn** [mesosphere/marathon#2203]
As suggested by @okuryu in #2198.
Github will soon enable protected branches: https://github.com/blog/2051-protected-branches-and-required-status-checks

We should protect master in `marathon` and `marathon-ui` as soon as this is available.

**Comment by okuryu** [comment]
:+1:

**Comment by aldipower** [comment]
I was able to protected these branches in Marathon UI:
```
master
```
```
DO-NOT-DELETE-0.9.0-and-0.9.1-assets
```
```
fix/assets-in-0.9.1-release
```
This prevents deleting and force push.
